ddpui/ddpairbyte/airbyte_service.py

```diff
@@ -301,6 +301,26 @@ def get_sources(workspace_id: str) -> List[Dict]:
     return res
 
 
+def get_sources_v1(workspace_id: str, limit: int, offset: int, search: str = "") -> List[Dict]:
+    """Fetch paginated sources from airbyte workspace with optional search"""
+    if not isinstance(workspace_id, str):
+        raise HttpError(400, "Invalid workspace ID")
+
+    payload = {
+        "workspaceId": workspace_id,
```

Comment:
can you check the api spec properly ?
https://airbyte-public-api-docs.s3.us-east-2.amazonaws.com/rapidoc-api-docs.html#post-/v1/sources/list_paginated

the payload requires  `workspaceIds`, in our case it will be a list of length 1